test/runtests.jl

```diff
@@ -1,315 +1,556 @@
 using Base.Test, ODBC, DataStreams, Missings, WeakRefStrings, DataFrames, DecFP
 
+# To run these tests first run these statements from the command line in the tests directory:
+# odbcinst -i -s -h -f mytest.odbc.ini
+# odbcinst -i -s -h -f pgtest.odbc.ini
+# odbcinst -i -s -h -f ./test/mssqltest.odbc.ini
+
+# You can modify the above files as needed to point to your mysql, postgres, or mssql
+# database and drivers, or to specify your username and password
+# Note:
+# Microsoft drivers don't support reading usernames and passwords from .odbc.ini files so
+# you have to specify those in the DSN setup call:
+# ODBC.DSN("MSSQL-test", "yourusername", "yourpassword")
+
+TEST_MYSQL = true
+TEST_POSTGRESQL = true
+TEST_MSSQL = true
+
 @show ODBC.drivers()
 @show ODBC.dsns()
-
 @show ODBC.API.odbc_dm
-
 @show run(`odbcinst -q -d`)
-
 run(`uname -a`)
 
-dsn = if haskey(ENV, "ODBC_DSN")
-    ODBC.DSN(ENV["ODBC_DSN"])
-else
-    props = open(f->Dict(split(ln, ':') for ln in eachline(f)) , "/Users/jacobquinn/.tug/application.properties")
-    RIG, server, port = map(strip, split(props["environment.subdomain"], '.'))[1], strip(props["routing.services.mysql.host"]), strip(props["routing.services.mysql.port"])
-    ODBC.DSN("Driver={MySQL ODBC Driver}; user=domo; password=popchart; server=$server; port=$port;")
+@testset "mysql" begin
+    dsn = try
+        dsn = ODBC.DSN("MySQL-test")
+    catch e
+        println(STDOUT, "error connecting to mysql: ", e)
+        TEST_MYSQL = false
+    end
```

Comment:
Hmmm, I think wrapping the `if TEST_MYSQL` around the "mysql" testset would be better. In the scenario where the DSN isn't setup and `TEST_MYSQL == true` I think throwing an exception is fine as the testset should capture it and it'll just cause the "mysql" testset to error only (I think)